lib/runner.nix

```diff
@@ -40,11 +40,12 @@ let
   binScripts = microvmConfig.binScripts // {
     microvm-run = ''
       set -eou pipefail
+      unset EXTRA_ARGS
```

Comment:
Why do we need to unset EXTRA_ARGS here?

Also we should probably document this somewhere.

Reply:
> Why do we need to unset EXTRA_ARGS here?

We unset it just in case a variable with the same name was set earlier somewhere else so it doesn’t mess up the runner command.

> Also we should probably document this somewhere.

Sure, I updated the option description.